src/SignalR/clients/csharp/Client.SourceGenerator/src/Microsoft.AspNetCore.SignalR.Client.SourceGenerator.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <LangVersion>9</LangVersion>
+        <IncludeBuildOutput>false</IncludeBuildOutput>
+        <SuppressDependenciesWhenPacking>false</SuppressDependenciesWhenPacking>
+        <NoPackageAnalysis>true</NoPackageAnalysis>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis.Analyzers" PrivateAssets="all" />
+    <Reference Include="Microsoft.CodeAnalysis.CSharp" PrivateAssets="all" />
+    <Reference Include="Microsoft.AspNetCore.SignalR.Client.Core" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="$(OutputPath)\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
```

Comment:
This file needs some scrutiny. I'm not sure if:
1.  `<Reference .. />` tags support `PrivateAssets` property.
2.  `<None .. />` tag works for how ASP.NET Core is packaged and distributed.

Reply:
><Reference .. /> tags support PrivateAssets property.

Should be supported. We'll double check once ready to merge.

> `<None .. />` tag works for how ASP.NET Core is packaged and distributed.

Not sure, but again, will double check when ready to merge.

We also probably need to set a property or 2 to mark the package as shipping, but can do that later.

Reply:
> > `<Reference .. />` tags support `PrivateAssets` property.

> Should be supported. We'll double check once ready to merge.

`<Reference />` supports all `<PackageReference />` or `<ProjectReference />` metadata. Well, the subset for the item type we convert `<Reference />` into. This will work.

> We also probably need to set a property or 2 to mark the package as shipping, but can do that later.

This shouldn't be necessary but it's difficult to confirm in PR validation because we (intentionally) don't upload artifacts when building from a fork.